package.json

```diff
@@ -106,6 +107,7 @@
     "jest-extended": "^0.11.2",
     "jest-fetch-mock": "^2.1.0",
     "jest-react-native": "^18.0.0",
+    "jetifier": "^1.6.5",
```

Comment:
Do you use this?

Reply:
I used it, yes. `react-native-permissions` already uses AndroidX, so I had to reverse it.

Reply:
And we will probably have to use it more often in the future.